contributions/catalog/requirements.txt

```
@@ -1,4 +1,6 @@
 Flask==1.1.1
+itsdangerous==2.0.1
+Jinja2==3.0.3
 pymongo==3.7.2
```

```suggestion
werkzeug==2.0.2
```
Add this requirement as well.